Makefile

```diff
@@ -149,7 +149,7 @@ endif
 	install support/c/${IDRIS2_SUPPORT} ${PREFIX}/lib
 	mkdir -p ${PREFIX}/bin/${NAME}_app
 	install ${TARGETDIR}/${NAME}_app/* ${PREFIX}/bin/${NAME}_app
-	-rm -f "${PREFIX}/bin/${NAME}_app"/*.ss  # SS files interfere with separately compiled idris
+	$(RM) -f "${PREFIX}/bin/${NAME}_app"/*.ss  # SS files interfere with separately compiled idris
```

Comment:
The default value for the `RM` variable is `rm -f`.  And you have missed `-` before `$(RM)`.

Reply:
Other occurrences of `$(RM)` don't use `-` (for example in the `clean` target) so I removed it. Should I put it back in?

Reply:
If your intention was to continue even if it is unable to remove files then you should put `-` back.